### DISCUSSION TITLE
Is there a way to send feedback on GPT's Response to GPT. Like we have thumbsup and thumbs down buttons in chat.openai.com

### DISCUSSION THREAD

**Discussion by praneetnadkar** [microsoft/semantic-kernel#2665] [General]
<img width="706" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/7333796/af19bb00-2a38-4857-9b8d-ec643d8607de">

Is there a way to send this feedback to GPT like we can do in the Chat GPT UI?

**Comment by craigomatic** [comment]
There is a related item on the Chat-Copilot backlog: https://github.com/microsoft/chat-copilot/issues/42

To implement something similar you would end up keeping a collection of "good" examples (thumbs up) and "bad" examples (thumbs down), providing them to the model as few-shot examples with each request. The model itself is stateless.
